metadata_mapper/validate_mapping.py

```diff
@@ -1,11 +1,22 @@
-import sys
-import os
 import boto3
-import requests
 import json
-import settings
 import logging
+import os
+import requests
+import sys
 import urllib3
+
+from datetime import datetime
+from pathlib import Path
+from typing import Type, Union
+
+import settings
+import utilities
+
+from map_registry_collections import MAPPER_MAP
```

Comment:
`lookup`? should really switch to using `rikolti_mapper_type` rather than `mapper_type` + `lookup` (or in this case, `mapper_type` + `MAPPER_MAP`